### DISCUSSION TITLE
Arrow line shapes

### DISCUSSION THREAD

**Discussion by alfiesong** [tone-row/flowchart-fun#58] [Features / Syntax]
Love it!
If arrow shapes could be changed it will be great! currently all arrow lines are a straight, an elbow shape would help make the chart organized and no line cross.

**Comment by rob-gordon** [comment]
@alfiesong Sorry for the late response!

I think
```yaml
style:
  - selector: edge
    style:
      curve-style: taxi
```
is what you're looking for. 

Here's an example:
https://flowchart.fun/c#H4hQGcBcE8BsFMBcoAEKC0LzwQY0gPYBOiK8AJgObyppYwLJ124CuRAbvOlHEipACGADwCWoEMFAAVABajwKQQAdlKAO7EA1ooBG0AdGWiAdpVoBJE+XgnIpyilxF4gyPEWCUsU1oEEBWXgndhc7b1MaNEETA3dhSFJdeAAzYmCvXAJYAhMnFzcPJW9BZNhaAGEC92KfEz9yURd8WANWcAdA4PhhQXwSsoEeyFo0Hy1gyHlwUgAKOQUlVQ1tPTijBwBKUZQAbTYoAgBbFAsAEQBdFGIdtH0lckazJTzd86uY8hQXFPgw3E6Uzctzo40m0zmB0IJ3OmxQAHp4SgAJoEVhOGJKWDgALtYIdMwIdB1YJZI5HWyQcCgeEAKlANyOrFg9hJoDJFLs1NAAAlBFwUClWCYAIQoQA8G4BI-dAtPhQA
